src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,275 @@
+
+### Conversion
+
+By standing while conscious near people, you increase their Influence to your control. Once they reach a certain number, you can target them with your innate Convert ability. This mentally dominates a loyal crewmember and hands them their shiny Rev antag token. They are not told who converted them; it’s your choice as Head Rev to communicate this information.
+
+### Centcomm Uplink (“The Station Charter”)
+
+One of the consoles on the Bridge contains the Station Charter. This is the symbolic link between Centcomm and the station and allows Centcomm control over core security things like “where is the space station?” and “how is the space station” and “who is running the space station”.
+
+This Charter can be changed by consensus of the Heads. Doing this in a normal round risks a Deathsquad. But in Revs, triggering this triggers the end of the round and the Revs winning.
+
+To trigger this:
+
+1. Get all the Heads ID cards. This must be their special ID cards, not HOP-manufactured ones.
+
+2. Input them into the console
+
+3. Click the button.
+
+4. Wait 90 seconds for the uplink to be disabled. This alerts the entire station over comms that the Revs are doing this: I hope you have fortifed the bridge!
+
+# Role Outline: Revolutionary
+
+## How It Should Feel
+
+Plucked from your normal life by sudden desires for mutiny, you need to spread the good word across the station and assist your friends in overthrowing Centcomm. But if anyone takes a proper look at you, the game’s up, and you’ll be hauled over to Security… or worse.
+
+## Goals
+
+### Spread The Revolution
+
+Use Propaganda posters and literature to spread the Influence of the Revolution and get into the harder-to-reach areas of the ship.
+
+### Free Yourself From Tyranny
+
+Ambush and arrest your department head; steal their ID for the greater good, and requisition their tools for the Revolution.
+
+### Don’t Get Gibbed By Sec
+
+Avoid being rumbled by Sec; they have guns and equipment that makes rumbling and smashing your mutiny possible, and it’s hard (although not impossible) to flip Sec members to your side.
+
+Core Mechanics
+
+### Propaganda
+
+The printers, posters and books of the station can be influenced by Revolutionaries, who can also make them via crafting and using the station’s printers. Sec, the HoP and the Captain are trained to identify revolutionary literature (it’s not obvious to the rest of the station), but it’s also possible for traitors and nukies to make them to throw Sec off their own activities.
+
+Propaganda slowly ticks up an exposed character’s Influence meter and allows Head Revs to quickly convert a compromised department if not dealt with.
+
+### Drugs
+
+Mindbreaker Toxin can be manufactured and administered as a bulk Influence dose.
+
+### Brainwashing
+
+Converted Revolutionaries have an obvious trait that can be inspected:
+
+“They appear slightly dazed and confused”.
```

Comment:
With how many mentions there have been with metagaming, I feel it necessary to point this out: if someone is seen with that inspect status, they WILL be metagamed by someone at some point. It is pretty much inevitable. It is the same case of how it is now; security sees a passenger with a flash, they immediately call revolutionaries. Even if a security member did not call a revolution early (based on the sighting of this status), they certainly would if they see many people with the same status.

To address it being hidden: it's a neat idea, but that alone is not going to stop metagaming from occurring.

Reply:
I agree; it's currently the best idea for a drawback to being a Rev (that you're identifiable as valid by Security) I can think of.

There needs to be some drawback to being a Rev, otherwise people will go "oh, it's a Rev round, I'll be safe if I stand next to this poster then wander over to that particularly sus clown". Being valid is the best idea I have so far.

Reply:
not just by security but literally the entire station